src/main/java/com/rackspacecloud/metrics/ingestionservice/listeners/rolluplisteners/RollupListener.java

```diff
@@ -144,16 +145,15 @@ private void writeToInfluxDb(
                         payload, tenantIdAndMeasurement.getTenantId(),
                         tenantIdAndMeasurement.getMeasurement(), rollupLevel);
 
-            } catch (Exception e) {
+            } catch (IOException e) {
                 String msg = String.format("Write to InfluxDB failed with exception message [%s].", e.getMessage());
                 if(e.getCause().getClass().equals(ResourceAccessException.class)){
                     log.error(msg, e);
-                }
-                else {
+                } else {
                     log.error("[{}] Payload [{}]", msg, payload, e);
                 }
 
-                throw new Exception(msg, e);
+                throw new IOException(msg, e);
```

Comment:
Could use suggested `IngestFailedException` to wrap the cause and message.

Reply:
Huh it didn't suggest that to me and doesn't even think that it exists in a particular package for me.

Reply:
Sorry, I meant the exception I suggested out of thin air further up in the PR comments :)

Reply:
InvalidDataException?

Reply:
It seems like at this point, something more general would have failed like network connectivity, which is why I suggested `IngestFailedException`, which would be a parent class of `InvalidDataException`.